packages/react/src/components/LinearGradient/LinearGradient.stories.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { Tile } from '../Tile';
+import { TextInput } from '../TextInput';
+import './linear-gradient-story.scss';
+
+export default {
+  title: 'Experimental/unstable__LinearGradient',
```

Comment:
I'm not sure about having this in the storybook long term. Feels more like something we should find a way to document in the Themes elements example. Not a blocker for this PR, but something to consider for the future.

Reply:
Yeah I can remove this before we merge 👍🏻